credits.txt

```diff
@@ -288,28 +316,32 @@ contributed to Endless Sky:
   thomasballinger
   ThrosturX
   Thunderforge
+  tibetiroka
   tiennou
   timbennett
   TJesionowski
   tmbutterworth
   toilethinges
+  TotalCaesar659
   TurkeyMcMac
   Turtleroku
   tux2603
-  Unjown
+  unjown
```

Comment:
At the risk of making this list impossible to maintain: If we're including typo fixes, we should probably include "indirect" contributions such as #5921
```suggestion
  UnciaPrima
  unjown
```

Reply:
I'm really not sure how to go about supporting this. I like the idea, but I've lost count of how many typo fixes I've submitted (and seen others submit) that were reported by others. Trying to include all of those will be a long project. If we do that, I'd say that should be a different PR that tries to get all of them in one go rather than just including one or two in this.